app/src/features/charts/DisparityBarChart.tsx

```diff
@@ -211,19 +215,23 @@ function TwoVarBarChart(props: {
   breakdownVar: string;
   breakdownVarDisplayName: string;
 }) {
+  const [ref, width] = useResponsiveWidth();
   return (
-    <Vega
-      spec={getSpec(
-        props.data,
-        props.breakdownVar,
-        props.breakdownVarDisplayName,
-        props.thickMeasure,
-        props.thickMeasureDisplayName,
-        props.thinMeasure,
-        props.thinMeasureDisplayName
-      )}
-    />
+    <div ref={ref as RefObject<HTMLDivElement>}>
```

Comment:
Is it possible to bake the return type into the useResponsiveWidth so it doesn't require casting?

Reply:
oh ha I actually had already done that I guess I forgot to remove this cast